src/entities/exit/poolExit.ts

```diff
@@ -18,6 +19,7 @@ export class PoolExit {
         const { customPoolExits } = config || {};
         this.poolExits = {
             Weighted: new WeightedExit(),
+            PHANTOM_STABLE: new ComposableStableExit(),
```

Comment:
Can you add a comment (same as Join) that explains the `PHANTOM_STABLE` naming as its a bit confusing and useful for reference.